ppr-ui/src/views/newMhrRegistration/HomeOwners.vue

```diff
@@ -88,9 +88,11 @@
         Add a Business or Organization
       </v-btn>
       <div class="my-6">
-        <span class="generic-label">Home Tenancy Type: </span>{{ getHomeTenancyType() }}
+        <div><span class="generic-label">Home Tenancy Type: </span>{{ homeTenancyType }}<span v-show="showGroups"><span v-show="ownershipAllocation.hasMinimumGroupsError" class="error-text fs-14 ml-3"
+            >Must include more than one group of owners</span
+          ></span></div>
         <div v-show="showGroups">
-          <span class="generic-label">Total Ownership Allocated: </span>{{ ownershipAllocation.totalAllocation }}
+          <span class="generic-label">Total Ownership Allocated:</span><span> {{ interestType }} </span>{{ ownershipAllocation.totalAllocation }}
```

Comment:
A newline never hurts for readability, with multiple tags/computed values etc.

Reply:
agreed.. fixed..